Sources/SwiftParser/Patterns.swift

```diff
@@ -274,6 +289,9 @@ extension Parser.Lookahead {
       case `var`
       case leftParen
       case `inout`
+      case `mutating`
+      case borrowing
+      case consuming
```

Comment:
Same comment of using `ValueBindingPatternSyntax.BindingSpecifierOptions` applies here.